### DISCUSSION TITLE
Which JSON schema version is `pydantic` compliant with?

### DISCUSSION THREAD

**Discussion by jamesbraza** [pydantic/pydantic#6237] [Question] [answered]
I am trying to use a `BaseModel.schema()` in PyCharm as a JSON schema.

Which schema version does `pydantic` use?  Version 4, 6, or 7?

![image](https://github.com/pydantic/pydantic/assets/8990777/7f642e42-2244-4f2f-926f-f9232f591f2c)

Also, I would gladly open a docs PR with this information.

**Comment by samuelcolvin** [comment, marked as answer]
Pydantic V2 is compliant with JSON Schema 2020-12.

Pydantic v1 isn't strictly compliant with a specific version, but I believe even version 7 is very old, so probably use that.

**Reply by jamesbraza** [reply]
Thank you for the speedy answer!  I realized version 7 is from 2017 (6 years ago), so you're right that it's old. https://youtrack.jetbrains.com/issue/IDEA-275297 is a PyCharm ticket from 2021 requesting support for newer versions.
